FINGERPRINT_Toolbox_Inventory.adoc

```diff
@@ -24,19 +24,29 @@ Certain commercial equipment, instruments, or materials are identified in this t
 |===
 |ID
 |Category
-|Requirement
+|Requirements
 |Commercial Examples
 
 
-|T.1
-|Commercial fingerprint scanner
-a|Commercial fingerprint scanner
+.2+|T.1
+.2+|Commercial fingerprint scanner
+a|Scanner must meet one of the following (or equivalent) qualifications:
+
+* [ISO/IEC 19794-4:2011] Information technology — Biometric data interchange formats — Part 4: Finger image data
```

Comment:
Add the link.

```suggestion
* https://www.iso.org/standard/50866.html[ISO/IEC 19794-4:2011] Information technology — Biometric data interchange formats — Part 4: Finger image data
```

Reply:
I had originally left out the URL since it is basically the purchase site, but that is fine given we are providing the links to the others.

Reply:
@woodbe either way. Should we remove the brackets if there's no link?

Reply:
I just added the link. I added the brackets only because that is how we referenced ISO specs in the other docs, they always had brackets on them. I guess we could add them on the other docs to be consistent.